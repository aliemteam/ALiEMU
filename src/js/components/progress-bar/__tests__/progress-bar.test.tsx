import React from 'react';
import renderer from 'react-test-renderer';

import ProgressBar from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<ProgressBar value={5} max={10} />);
    expect(tree).toMatchSnapshot();
});
