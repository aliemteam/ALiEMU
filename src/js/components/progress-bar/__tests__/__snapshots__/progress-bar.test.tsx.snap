// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  aria-valuemax={10}
  aria-valuemin={0}
  aria-valuenow={5}
  role="progressbar"
>
  <div
    style={
      Object {
        "width": "50%",
      }
    }
  />
</div>
`;
